src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -294,7 +293,23 @@ std::pair<SQLPipelineStatus, const std::shared_ptr<const Table>&> SQLPipelineSta
 
   // Get output from the last task if the task was an actual operator and not a transaction statement
   if (!_is_transaction_statement()) {
-    const auto& last_executed_operator = static_cast<const OperatorTask&>(*tasks.back()).get_operator();
+    if constexpr (HYRISE_DEBUG) {
```

Comment:
Please doc this block (296-312). *Why* are we looking for that one operator that is `ExecutedAndAvailable`?